src/components/People.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import axios from 'axios'
+
+import Person from './Person'
+
+export class People extends React.Component {
+    constructor(){
+        super()
+        this.state={
+            person: '',
+            personInfo:[],
+            loadData: false
+
+        }
+    }
+
+    personName = (e) =>{
```

Comment:
I suggest using a more descriptive name like handleNameInput that explains what the function does.